### DISCUSSION TITLE
P1001: Can't reach database server at `ec2-34-242-89-204.eu-west-1.compute.amazonaws.com`:`5432`

### DISCUSSION THREAD

**Discussion by bytesagar** [prisma/prisma#10482] [Q&A]
I can connect to the database from my terminal but prisma refuses to connect.

**Comment by austincrim** [comment]
Hey @bytesagar 👋,

When you say that Prisma can't connect, are you running locally or within a deployed app? Are you sure you have the same environment variables available in both scenarios?

**Reply by bytesagar** [reply]
@austincrim Yes i am running it locally and the database is hosted in cloud. I was trying to use it in local. But i couldnt, it throws error everytime.
Should i host the api and try again.

**Reply by janpio** [reply]
Did you confirm that the server is configured to accept requests from external sources?
Can you connect to the same connection string with other tools from your computer?

**Reply by bytesagar** [reply]
@janpio 
Yes i have already tried it in cli it is connecting without any errors

**Reply by janpio** [reply]
If you can connect manually from the same computer, you should be able to connect with Prisma as well. Are you sure the full connection string is correct?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
